includes/Stories_Renderer/Renderer.php

```diff
@@ -462,8 +486,11 @@ public function render_single_story_content() {
 		$single_story_classes = $this->get_single_story_classes();
 		$lightbox_state       = 'lightbox' . $this->current()->get_id();
 
-		// Web Stories Styles for AMP and non-AMP pages.
-		wp_enqueue_style( self::STYLE_HANDLE );
+		// No need to load these styles on admin as editor styles are being loaded by the block.
+		if ( ! is_admin() ) {
+			// Web Stories Styles for AMP and non-AMP pages.
+			wp_enqueue_style( self::STYLE_HANDLE );
+		}
```

Comment:
Can you explain this change a little more. I find it confusing

Reply:
As mentioned in the comment, the styles needed for the block to render properly on the editor is already being loaded by the Block class. 
This was adding duplicate styles on the admin screens as well. And we need this stylesheet to only load on front end.

And since this gets loaded on render, when the user first inserts the block they were seeing the broken block previously, hence imported front end style in editor styles for the block.